sync/downstream.py

```diff
@@ -1069,3 +1075,32 @@ def update_pr(git_gecko, git_wpt, sync, action, merge_sha, base_sha, merged_by=N
     except Exception as e:
         sync.error = e
         raise
+
+
+def get_temporary_metadata(sync):
+    pr_id = sync.pr
+    _, statuses = env.gh_wpt.get_combined_status(pr_id)
+    for status in statuses:
+        if status.context == "Taskcluster (pull_request)":
```

Comment:
Hmm, so this name might change if we ever switch on the checks API (which we previously tried to do and failed because wpt.fyi wasn't ready). I wonder if we can look for things with a taskcluster URL (although that will also change; we could/should make it a config variable).

Reply:
I've added a config under `[web-platform-tests]` `ci.context` in the `sync.ini` files